lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -28,69 +28,20 @@
-                            <nav.bordered-section>
-                                {{#if draftManager.prevPageParam}}
-                                    <nav.links.default
-                                        data-test-goto-previous-page
-                                        data-analytics-name='Sidenav back'
-                                        @route='registries.drafts.draft'
-                                        @models={{array draftManager.draftId draftManager.prevPageParam}}
-                                    >
-                                        <nav.icon @icon='arrow-left' />
-                                    </nav.links.default>
-                                {{/if}}
-                                {{#if draftManager.nextPageParam}}
-                                    <nav.links.default
-                                        data-test-goto-next-page
-                                        data-analytics-name='Sidenav next'
-                                        @route='registries.drafts.draft'
-                                        @models={{array draftManager.draftId draftManager.nextPageParam}}
-                                        @onClick={{draftManager.setPageIsVisited}}
-                                    >
-                                        <nav.icon @icon='arrow-right' />
-                                    </nav.links.default>
-                                {{/if}}
-                                {{#if (eq draftManager.lastPage this.pageIndex)}}
-                                    <nav.links.default
-                                        data-test-goto-review
-                                        data-analytics-name='Sidenav review'
-                                        @route='registries.drafts.draft'
-                                        @models={{array draftManager.draftId 'review'}}
-                                    >
-                                        <nav.icon @icon='arrow-right' />
-                                    </nav.links.default>
-                                {{/if}}
-                                {{#if this.inReview}}
-                                    <Drafts::Draft::-Components::Register
-                                        @draftRegistration={{this.draftRegistration}}
-                                        @draftManager={{draftManager}}
-                                        @onSubmitRedirect={{action this.onSubmitRedirect}}
-                                        @showMobileView={{this.showMobileView}}
-                                    />
-                                {{/if}}
-                            </nav.bordered-section>
-                        </Navbar>
-                    {{/if}}
                 </layout.heading>
+                {{#if this.showMobileView}}
+                    <layout.top
+                        @inReview={{this.inReview}}
+                        @draftManager={{draftManager}}
```

Comment:
So `osf-layout/top-nav` is very specific to this page -- even down to using `draftManager`. Seems like it'd make more sense as a route-local component, like `drafts/draft/-components/top-nav`.

Or, maybe better but more work, separate it into a generic, empty `osf-layout/top-nav` and a route-local component that fills it with draft navigation controls?

Reply:
IMHO, we should do the latter (that's what I was expecting :wink:).

Reply:
Also, I would just call this the osf-layout component `top` since it won't necessarily be a nav.